__std__/cells/automation/ciJobs.nix

```diff
@@ -41,9 +56,14 @@ let
 
   jobs =
     if system == "x86_64-linux" then
-      { mingwW64 = windows-plutus-jobs; } // native-plutus-jobs // other-jobs
+      { mingwW64 = windows-plutus-jobs; } //
+      native-plutus-jobs //
+      { ghc924 = native-plutus-924-jobs; } //
+      other-jobs
     else if system == "x86_64-darwin" then
-      native-plutus-jobs // other-jobs
+      native-plutus-jobs //
+      native-plutus-924-jobs //
```

Comment:
we want `{ ghc924 = native-plutus-924-jobs; }` here

Reply:
argument for deduplicating this and writing
```
jobs = x // y // lib.optionalAttrs (system == "x86_64-linux") { mingwW64 = ... } // ...
```

Reply:
👍